bokehjs/src/coffee/core/bokeh_events.coffee

```diff
@@ -0,0 +1,96 @@
+# Granular event classes for use with the event system
```

Comment:
I would like this to be actually moved to `core/`, especially that `core/ui_events` is the only module where this is imported.